src/tests/functional/plugin/cpu/single_layer_tests/gru_cell.cpp

```diff
@@ -166,7 +166,11 @@ const std::vector<std::vector<ov::test::InputShape>> dynamicShapes = {
     { { { {1, 10}, {25, 35} },             // Dynamic shape 0
         { {2, 30}, {5, 30}, {8, 30} } },   // Target shapes
       { { {1, 10}, -1 },                   // Dynamic shape 1
-        { {2, 10}, {5, 10}, {8, 10} } } }  // Target shapes
+        { {2, 10}, {5, 10}, {8, 10} } } }, // Target shapes
+    { { { {1, 10}, {25, 35} },             // Dynamic shape 0
+        { {2, 30}, {2, 30}, {5, 30}, {5, 30}, {8, 30} } },   // Target shapes
```

Comment:
Even though it is not obvious, but placing the same target shapes one after another does not properly test the cache. The reason is that we have yet another optimization for this case. If the inputs shapes for a node are not changed we assume that we can reuse its previous state, that means we do not call shape infer and parameters preparation. To test the cache properly we have to interleave the shapes, so that the aforementioned optimization is not used, e.g. `{2, 30}, {5, 30}, {2, 30}, {5, 30}, {8, 30}`. Could you please check all the related tests definitions?

Reply:
Done